source/blog/2010-12-21-doctrine2-released.md

```diff
@@ -163,7 +163,7 @@ Where do I start?
 You can download Doctrine 2 [from our downloads
 section](https://www.doctrine-project.org/projects/orm/download) ,
 [install it via PEAR](http://pear.doctrine-project.org/) or find it in
```

Comment:
PEAR isn't used anymore.

Reply:
delete it?

Reply:
I left it as is because of https://github.com/doctrine/doctrine-website/pull/279#discussion_r246491823

Reply:
I'm not sure if we should keep dead links. It makes people open PRs or issues about that. At least this should be addressed as "kept for historical purpose" to let readers know this is intentionally kept this way.

Reply:
need @jwage here

Reply:
My vote would be to leave it, we can't realistically remove and rewrite every blog post from a decade ago where links stop working. We are way over complicating this PR and thinking way too much about it. Just replace the things where it makes sense and lets merge this PR and move on.